src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2707,27 +2732,13 @@ private void EmitConstantExpression(TypeSymbol type, ConstantValue constantValue
         private void EmitInitObj(TypeSymbol type, bool used, SyntaxNode syntaxNode)
         {
             if (used)
-            {
-                if (type.IsPointerType() || type.SpecialType == SpecialType.System_UIntPtr)
```

Comment:
Will add "PROTOTYPE" for this. There are several "PROTOTYPE" in codegen already so this one would be addressed together with others.

---
In reply to: [118798124](https://github.com/dotnet/roslyn/pull/19713#discussion_r118798124) [](ancestors = 118798124,118782391)